components/InventoryItemListItem/index.tsx

```diff
@@ -160,9 +186,33 @@ export default function InventoryItemListItem({
         {typeof inventoryItem.assignee === 'string'
           ? inventoryItem.assignee
           : inventoryItem.assignee.name}
-        <IconButton sx={{ ml: 'auto' }}>
-          <MoreVert sx={{ color: theme.palette.grey['500'] }} />
-        </IconButton>
+        <Box sx={{ flexGrow: 0, ml: 'auto' }}>
+          <IconButton onClick={handleOpenKebabMenu}>
+            <MoreVert sx={{ color: theme.palette.grey['500'] }} />
+          </IconButton>
+          <Menu
+            sx={{ mt: '45px' }}
```

Comment:
try use `mt: 2` or some other number like that instead of pixel values here. if you need to use the pixel values that's fine but start with the preset values

Reply:
Done!